apps/prairielearn/package.json

```diff
@@ -78,37 +78,37 @@
     "crypto-js": "^4.2.0",
     "csv-parse": "^5.5.5",
     "csvtojson": "^2.0.10",
-    "d3": "^7.8.5",
+    "d3": "^7.9.0",
     "date-fns": "^2.30.0",
-    "date-fns-tz": "^2.0.0",
+    "date-fns-tz": "^2.0.1",
     "debug": "^4.3.4",
     "delegated-events": "^1.1.2",
-    "dockerode": "^3.3.5",
-    "dompurify": "^3.0.9",
+    "dockerode": "^4.0.2",
```

Comment:
I couldn't figure out what the breaking changes were here, but I did see the code change when calling `docker.listContainers()`.

Reply:
Follow breadcrumbs from here: https://github.com/apocas/dockerode/issues/751

As for the `listContainers` change, that was a genuine bug uncovered by the introduction of better types!

Reply:
Oh nice! 🙏 for fixing this bug upstream. And yay for types!